### PR TITLE
[tests] add ping test using LLA source

### PR DESCRIPTION
**Pull request by simonlingoogle**

### CODE REVIEW

File: tests/scripts/thread-cert/test_ping_lla_src.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+        left_lla = left.get_ip6_address(config.ADDRESS_TYPE.LINK_LOCAL)
+
+        # Ping a neighbor with LLA src should succeed.
+        self.assertTrue(left.ping(center.get_mleid(), interface=left_lla))
+        self.simulator.go(3)
+
+        # Ping a non-neighbor with LLA src should fail.
+        self.assertFalse(left.ping(right.get_mleid(), interface=left_lla))
+        self.simulator.go(3)
```
Comment (superwhd) [inline tests/scripts/thread-cert/test_ping_lla_src.py:101]: I think it's important to verify the packet is not sent out in the `verify` stage.

Comment (simonlingoogle) [reply tests/scripts/thread-cert/test_ping_lla_src.py:101]: It's actually sent out. 

So, I prefer to not verify it but only verify that the ping eventually fails.

Comment (superwhd) [reply tests/scripts/thread-cert/test_ping_lla_src.py:101]: ACK

**Review by superwhd — APPROVED** [review]
LGTM :+1:

**Review by jwhui — APPROVED** [review]

**Pull request merged**